source/web/css/mei-screen.css

```diff
@@ -192,6 +191,7 @@
 }
 
 .specPage .classBox .classHeading .classDesc {
+	display: inline-block;
```

Comment:
I can see what this changes but I'd rather add o.5rem padding to `.specPage .classBox .classHeading` then `.classDesc` would always stay inline with `.classLabel` but no run into the right border when it is longish

Reply:
This is just to keep `classDesc` better separated from `classLabel`.
It changes breaks from this:

<img width="519" alt="image" src="https://user-images.githubusercontent.com/7693447/195020251-2c1a3acc-2c27-44b3-8c21-a72b4eb5d083.png">

to this:

<img width="517" alt="image" src="https://user-images.githubusercontent.com/7693447/195020394-9e5af0e1-0ebe-4912-80fc-dff94008f2e1.png">

Reply:
This is true only for those `classDesc` that are too long to fit into the line.

Reply:
Ok, seems we cannot agree on this, so I removed it again.